hadoop-cloud/pom.xml

```diff
@@ -28,11 +28,9 @@
 
   <artifactId>spark-hadoop-cloud_2.12</artifactId>
   <packaging>jar</packaging>
-  <name>Spark Project Cloud Integration through Hadoop Libraries</name>
+  <name>Spark Project Hadoop Cloud Integration</name>
```

Comment:
I'm changing this file mostly to test the PR builder. But also to remove some verbosity...